admin/plugins.php

```diff
@@ -0,0 +1,399 @@
+<?php
+/**
+ * Admin settings helper functions.
+ *
+ * @package performance-lab
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Get info about all the standalone plugins and their status.
```

Comment:
Function descriptions should start in a third-person singular verb (unless they don't start with a verb at all, which sometimes can be the case).
```suggestion
 * Gets info about all the standalone plugins and their status.
```